receiver/nopreceiver/internal/metadata/testdata/config.yaml

```diff
@@ -0,0 +1,9 @@
+default:
```

Comment:
out of curiosity, what is this file for?

Reply:
this is what gets generated when you run the metadata generator. It takes the metadata.yaml file from the component and generates a a few files in the metadata directory